package.json

```diff
@@ -31,7 +31,8 @@
   "dependencies": {
     "@tailwindcss/aspect-ratio": "^0.4.0",
     "@tailwindcss/forms": "^0.4.0",
-    "tailwindcss": "^3.0.24"
+    "tailwindcss": "^3.0.24",
+    "tailwind-children": "^0.5.0"
```

Comment:
I tried this person's other plugin (which we were using for SAA on TW v2 I believe). It worked well with v2, but broke in v3 (the nesting classes got all messed up) so I ended up uninstalling it in My Account. That was back when TW v3 first came out - not sure if it's better now 🤷🏼‍♀️